packages/react-components/react-migration-v8-v9/src/stories/CheckboxShim/index.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+
+import descriptionMd from './Description.md';
+
+import { Checkbox } from '@fluentui/react';
+import {
+  FluentProvider,
+  Checkbox as CheckboxV9,
+  webLightTheme,
+  makeStyles,
+  shorthands,
+} from '@fluentui/react-components';
+import { CheckboxShim } from '../../components/Checkbox/index';
+
+const useStyles = makeStyles({
+  root: {
+    display: 'grid',
+    gridTemplateColumns: 'auto auto auto',
+    gridTemplateRows: '1fr',
+    width: 'fit-content',
+    alignContent: 'center',
+    alignItems: 'center',
+    justifyItems: 'center',
+    columnGap: '10px',
+    rowGap: '10px',
+  },
+  componentName: {
+    justifySelf: 'end',
+    ...shorthands.margin(0, '10px', 0, 0),
+  },
+});
+
+export const Default = () => {
+  const styles = useStyles();
+
+  return (
+    <div className={styles.root}>
+      <h3>v8</h3>
+      <h3>shim</h3>
+      <h3>v9</h3>
+      <Checkbox label="checkbox" required={true} />
+      <FluentProvider theme={webLightTheme}>
```

Comment:
The FluentProvider needs to be removed as we have a global FluentProvider already in storybook